lib/xdr/ledger_entries/asset_code12.ex

```diff
@@ -45,22 +45,24 @@ defmodule StellarBase.XDR.AssetCode12 do
   @impl true
   def decode_xdr!(bytes, term \\ nil)
 
-  def decode_xdr!(bytes, _term) do
-    {%XDR.FixedOpaque{opaque: code}, rest} =
+  def decode_xdr!(<<bytes::binary-size(12), rest::binary>>, _term) do
```

Comment:
@CristhianRodriguezMolina why is 12-byte pattern matching necessary here?

Reply:
@miguelnietoa This is in order to get the 12 bytes of the asset_code_12 when decoding the binary of an asset, this is to ensure that all the 12 bytes are off the rest of the binary and avoid errors when continuing with the rest of the decoding.